python/src/coupling_tools.py

```diff
@@ -251,7 +254,7 @@ def execute(cmd, cwd=None, shell=False, executable=None, hide_win=True,
         process.kill()
         stdout_data, stderr_data = process.communicate()
         raise RuntimeError('Command "' + cmd + '" times out after ' +
-                            str(timeout) + 's')
+                           str(timeout) + 's')
     returncode = process.poll()
 
     # check return code
```

Comment:
drop these changes too

Reply:
i never change this file, i don't understand